README.md

```diff
@@ -223,6 +223,133 @@ Starwhale is an MLOps platform. It provides **Instance**, **Project**, **Runtime
 
 👏 Now, you have completed the fundamental steps for Starwhale standalone.
 
+Let's go ahead and finish tutorial on the on-premises instance.
+
+## MNIST Quick Tour for on-premises instance
+![Create Job Workflow](docs/docs/img/create-job-workflow.gif)
+
+🍰 **STEP1**: installing minikube and helm
+- Minikube 1.25+
+- Helm 3.2.0+
+
+There are standard installation tutorials for [minikube](https://minikube.sigs.k8s.io/docs/start/) and [helm](https://helm.sh/docs/intro/install/) here.
+
+
+🍵 **STEP2**: Start minikube
+```bash
+minikube start
+alias kubectl="minikube kubectl --"
+```
+For users in the mainland of China, please add these startup parameters：--image-mirror-country='cn' --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers
```

Comment:
For users in the mainland of China, please add these startup parameters：`--image-mirror-country='cn' --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers` ?